libs/zzz/svgicons/src/icons/Slots/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './SlotFiveIcon'
```

Comment:
This is more of a nit, but I would prefer them to be named `Slot1Icon` since they are never represented by their letter presentation, and it would allow them to be sorted in the input alphabetically.